docs/studio-pro-plan.mdx

```diff
@@ -44,7 +44,7 @@ You can choose between monthly or yearly billing on all of our paid plans.
 
 ### Pro features as a team member
 
-If you are a team member of another user’s [team](teams.mdx), you also have access to [premium features](#pro-features)! However, if you want to create your own teams, you need to [upgrade to your own Team subscription](upgrade.mdx).
+If you are a team member of another user’s [team](teams.mdx), with the role of [Admin](teams.mdx#admin-role) or [Editor](teams.mdx#editor-role), you also have access to [premium features](#pro-features)! However, if you want to create your own teams, you need to [upgrade to your own Team subscription](upgrade.mdx).
```

Comment:
```suggestion
If you are a team member of another user’s [team](teams.mdx), with the role of [Admin](/docs/teams/#admin-role) or [Editor](/docs/teams/#editor-role), you also have access to [premium features](#pro-features)! However, if you want to create your own teams, you need to [upgrade to your own Team subscription](upgrade.mdx).
```

Docusaurus can struggle to resolve heading links that use the `.mdx` extension, so it’s easier to make them relative like this. (Yes, it’s incredibly annoying. I’m trying to work on some docs docs to help explain this better!)

Reply:
I really didn't understand when to use one method over another. Thanks for explaining that!